### PR TITLE
Update migration-engine-3.6-to-4.0.html.md

### PR DESCRIPTION
**Pull request by Freundschaft**
Fixes issue # (delete if not relevant)

Changes proposed in this pull request:

-

-

-

I confirm that this pull request was submitted according to the [contribution guidelines](/.github/CONTRIBUTING.md): (please @mention yourself to sign)

This pull request needs review by: (please @mention the reviewer if relevant)

added --provision-dwh-db in order to create db for dwh

### CODE REVIEW

**Comment by sandrobonazzola** [comment]
@didib can you please review?

File: source/documentation/migration-engine-3.6-to-4.0.html.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Install `ovirt-engine` package and dependencies:
 
 Copy your backup to the new system, and restore it with following command:
 
-    # engine-backup --mode=restore --no-restore-permissions --provision-db --file=engine-backup.tar.gz --log=engine-backup-restore.log
+    # engine-backup --mode=restore --no-restore-permissions --provision-db --provision-dwh-db --file=engine-backup.tar.gz --log=engine-backup-restore.log
```
Comment (didib) [inline source/documentation/migration-engine-3.6-to-4.0.html.md]: You might suggest to add this option only if dwh is set up in the 3.6 machine. Thanks.

Comment (Freundschaft) [inline source/documentation/migration-engine-3.6-to-4.0.html.md]: just made changes: https://github.com/oVirt/ovirt-site/pull/441/files#diff-3a0294bfbd8a14e4a9e07301b873558c

**Comment by sandrobonazzola** [comment]
@tiraboschi can you please review?

**Pull request merged**